matlab/JSBSimInterface.cpp

```diff
@@ -0,0 +1,954 @@
+#include "JSBSimInterface.h"
+#include <models/FGAircraft.h>
+#include <models/FGAccelerations.h>
+#include <math/FGQuaternion.h>
+
+/* 2021-03-21 compiles with JSBSim 1.1.5
+ * Compiles with JSBSim checkout a59596f8f0d4c7b4f7ba24e99997bc2071d6cb72
+ */
+
+JSBSimInterface::JSBSimInterface(FGFDMExec *fdmex)
+{
+	_ac_model_loaded = false;
+	fdmExec = fdmex;
+	propagate = fdmExec->GetPropagate().get();
+	accel = fdmExec->GetAccelerations().get();
+	accel->InitModel();
+	auxiliary = fdmExec->GetAuxiliary().get();
+	aerodynamics = fdmExec->GetAerodynamics().get();
+	propulsion = fdmExec->GetPropulsion().get();
+	fcs = fdmExec->GetFCS().get();
+	ic = new FGInitialCondition(fdmExec);
+	//verbosityLevel = JSBSimInterface::eSilent;
+}
+JSBSimInterface::JSBSimInterface(FGFDMExec *fdmex, double dt)
```

Comment:
```suggestion
JSBSimInterface::JSBSimInterface(double dt)
```
The instance of `FGFDMExec` is now a member of `JSBSimInterface`, there is no need to pass it as an argument.